build-system/tasks/serve.js

```diff
@@ -27,6 +27,18 @@ var useHttps = argv.https != undefined;
  * Starts a simple http server at the repository root
  */
 function serve() {
+  // Get the serve mode
+  if (argv.compiled) {
+    process.env.SERVE_MODE = 'min';
+    util.log(util.colors.green('Serving minified js'));
+  } else if (argv.cdn) {
+    process.env.SERVE_MODE = 'cdn';
+    util.log(util.colors.green('Serving current prod js'));
+  } else {
+    process.env.SERVE_MODE = 'max';
```

Comment:
But I think we are more familiar with `min`/`max`. And it's not obvious right now that `default` is `max` mode.